modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,39 +1,51 @@
 // Module included in the following assemblies:
 //
-// * cnv/cnv_users_guide/cnv-importing-vmware-vm-to-ocp.adoc
+// * cnv/cnv_users_guide/cnv-create-vms.adoc
+
 [id="cnv-importing-vmware-vm_{context}"]
-= Importing a VMware virtual machine or template
+= Importing a VMware virtual machine or template with the virtual machine wizard
+
+You can import a VMware virtual machine or template. If you import a template, a virtual machine based on the template is created.
 
-You can use the {ProductName} web console to import a VMware virtual machine or template. If you import a template, a virtual machine based on the template is created.
+:FeatureName: Importing a VMware virtual machine or template
+include::modules/technology-preview.adoc[leveloffset=+1]
+:!FeatureName:
 
 .Prerequisites
 
 * The VMware virtual machine is powered off.
 * The VMware Virtual Disk Development Kit (VDDK) has been uploaded to the namespace in which you are importing the virtual machine.
 
-.Procedure
+include::cnv-uploading-vddk.adoc[leveloffset=+1]
+
+== Importing the VMware virtual machine or template
 
 . Click *Workloads* -> *Virtual Machines* from the side menu.
 . Click *Create Virtual Machine* and select *Create with Wizard*.
 . Perform the following procedure in the *Basic Settings* screen:
 .. Click *Provision Source* and select *Import*.
 .. Click *Provider* and select *VMware*.
-.. Click *vCenter Instance* and select *Connect to New Instance* or an existing vCenter instance.
+.. Click *vCenter Instance* and select *Connect to New Instance* or a saved vCenter instance.
 +
 * If you select *Connect to New Instance*, provide values in the following fields:
 
 ** *vCenter Hostname*
 ** *vCenter User Name*
 ** *vCenter Password*
 +
-Optionally select *Save as New vCenter Instance* and click the 
-*Check* or *Check and Save* button. The wizard connects to the vCenter instance.
+Click *Check and Save*. The wizard connects to the vCenter instance and saves the credentials.
```

Comment:
I think it was already like this, but I think the indentation/lack of bullet is odd here. It's indented like it belongs under the third field of "vCenter Password", when it is actually another actionable step. Seems like it should have a bullet.

Reply:
If you think an option requires a separate bullet, I think this should be turned into a numbered subprocedure. Will change.